cmd/fissh/main.go

```diff
@@ -65,6 +67,16 @@ func main() {
 	}
 }
 
+func extractTimezoneFromEnv(env []string) (string, error) {
+	idx := slices.IndexFunc(env, func(s string) bool { return strings.HasPrefix(s, "TIMEZONE=") })
```

Comment:
feels like we should be able to use the environment as a `map[string] string` but im not familiar with this api so if this is the only way to use the environment then thats fine

Reply:
I agree, this felt gross but it appears this is the only format the env is provided in :(

> ```
> // Environ returns a copy of strings representing the environment set by the
> // user for this session, in the form "key=value".
> Environ() [][string](https://pkg.go.dev/builtin#string)
> ```
> \- https://pkg.go.dev/github.com/gliderlabs/ssh#Session